core/src/replay_stage.rs

```diff
@@ -250,7 +261,7 @@ impl ReplayStage {
 
         assert!(!poh_recorder.lock().unwrap().has_bank());
 
-        let (reached_leader_tick, grace_ticks, poh_slot, parent_slot) =
+        let (reached_leader_tick, _, poh_slot, parent_slot) =
```

Comment:
nit: My preference is to use `_grace_ticks` instead of `_` for unused tuple items.   This helps prevents an unreadable mess when more items are unused (eg,  `let (_, _, _, foo, _, _) = ..`)